src/Http/Http.Abstractions/src/ProblemDetails/IProblemDetailsService.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Http;
+
+/// <summary>
+/// Defines a type that provide functionality to
+/// create a <see cref="Mvc.ProblemDetails"/> response.
+/// </summary>
+public interface IProblemDetailsService
+{
+    /// <summary>
+    /// Write a <see cref="Mvc.ProblemDetails"/> response to the current context
```

Comment:
This type isn't really writing the ProblemDetails, maybe should mention it uses `IProblemDetailsWriter`s to try and write the response.